packages/cli/src/commands/dev.ts

```diff
@@ -14,6 +14,12 @@ import { resolvePath } from "../utils/parseNameAndPath";
 import { TriggerApi } from "../utils/triggerApi";
 import { run as ncuRun } from 'npm-check-updates'
 import chalk from "chalk";
+import path from 'path';
+import { createRequire } from 'node:module';
+import { PackageFile } from "npm-check-updates/build/src/types/PackageFile";
```

Comment:
@matt-aitken 
Built fine, since it's a type only import

<img width="239" alt="image" src="https://github.com/triggerdotdev/trigger.dev/assets/62795688/d33c7eef-a3bc-4816-b84f-ad76c9744cf8">